### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by C0fyC0de** [Chrisknyfe/water_shader#15]
Can you make a tutorial on how to set up a scene from zero with these assets?

**Comment by deaddogdown** [comment]
Yeah, I agree. It would be good to see a setup. I got issues that when I try change the waves parameters, the waves in the 3d editor don't update. Seeing a tutorial on how to change the wave parameters to preview in the 3d editor would be great. Really nice water but hoping you can give more detail of setup and tweaking.
